site/viamguides/src/plant-watering-device-workshop/plant-watering-device-workshop.md

```diff
@@ -46,50 +41,103 @@ Follow this workshop to learn how to set up an automatic plant watering system b
   - [a webcam (optional for the additional sections)](https://www.amazon.com/EMEET-1080P-Webcam-Microphone-Noise-Canceling/dp/B0CP6BTQKR)
 - Have a planter box or flower pot with soil or coffee grounds, a screw driver and a water container ready.
 
-### What You’ll Need
-
-- All the hardware components listed in prerequisites.
-- A [Python development environment](https://www.python.org) and [the Viam Python SDK](https://python.viam.dev)
-- A [GitHub](https://github.com/) account.
-- [VSCode](https://code.visualstudio.com/download) installed, or another similar code editor of your choice.
-
 ### Watch the Video
 
 Follow along with the step-by-step video.
 <video id="Q6UuUKJpDn0"></video>
 
 <!-- ------------------------ -->
+## Set up your Raspberry Pi
+Duration: 15
+
+The Raspberry Pi boots from a microSD card. You need to install Raspberry Pi OS on a microSD card that you will use with your Pi. For more details about alternative methods of setting up your Raspberry Pi, refer to the [Viam docs](https://docs.viam.com/installation/prepare/rpi-setup/#install-raspberry-pi-os).
+
+### Install Raspberry Pi OS
+
+1. Connect the microSD card to your computer.
+1. Launch the [Raspberry Pi Imager](https://www.raspberrypi.com/software/).
+   ![raspberry pi imager](assets/imager.png)
+1. Click **CHOOSE DEVICE**. Select your model of Pi, which is Raspberry Pi 4.
+1. Click **CHOOSE OS**. Select **Raspberry Pi OS (64-bit)** from the menu.
+1. Click **CHOOSE STORAGE**. From the list of devices, select the microSD card you intend to use in your Raspberry Pi.
+   ![raspberry pi storage](assets/imagerInitialSettings.png)
+1. Configure your Raspberry Pi for remote access. Click **Next**. When prompted to apply OS customization settings, select **EDIT SETTINGS**.
+   ![raspberry pi edit settings](assets/applyOSSettings.png)
+1. Check **Set hostname** and enter the name you would like to access the Pi by in that field, for example, `test`.
```

Comment:
This example hostname should match the screenshot of the Raspberry Pi Imager settings to avoid confusion:

```suggestion
1. Check **Set hostname** and enter the name you would like to access the Pi by in that field, for example, `echo`.
```